Projects/UOContent/Skills/Meditation.cs

```diff
@@ -71,7 +71,7 @@ public static TimeSpan OnUse(Mobile m)
             }
 
             var skillVal = m.Skills.Meditation.Value;
-            var chance = (50.0 + (skillVal - (m.ManaMax - m.Mana)) * 2) / 100;
+            var chance = (50.0 + ((skillVal - (m.ManaMax - m.Mana)) * 2)) / 100;
```

Comment:
I checked and I don't think this does anything.
<img width="570" alt="Screen Shot 2021-09-18 at 8 28 11 PM" src="https://user-images.githubusercontent.com/3953314/133914427-a9ad51ca-df2e-4f1b-907a-9ed1e8281909.png">

Note the gray parenthesis

Reply:
hmm ,I wonder why I edited it. ServUO have it with parenthesis but it doesnt change anything (only reading is better with it).